src/services/ui/src/pages/create/external-landing/ExternalAppLandingPage.tsx

```diff
@@ -0,0 +1,169 @@
+import { Button } from "@/components/Inputs";
+import { Link } from "react-router-dom";
+import { PropsWithChildren, ReactElement } from "react";
+import { SimplePageTitle } from "@/pages/create/create-options";
+import { SimplePageContainer } from "@/components";
+import { FAQ_SECTION, ROUTES } from "@/routes";
+export enum EXTERNAL_APP {
+  MAC_PRO = "https://www.medicaid.gov/resources-for-states/medicaid-and-chip-program-macpro-portal/index.html#MACPro",
+  MMDL = "https://wms-mmdl.cms.gov/MMDL/faces/portal.jsp",
+}
+
+export interface ExternalAppLandingPageConfig {
+  pageTitle: string;
+  image: ReactElement;
+  description: ReactElement;
+  buttonLabel: string;
+  buttonLink: EXTERNAL_APP;
+}
+const MACProLogo = () => (
+  <img src={"/images/logos/macpro.png"} alt={"MACPro system logo"} />
+);
+const MMDLLogo = () => (
+  <img src={"/images/logos/mmdl.png"} alt={"MMDL system logo"} />
+);
+const FAQHelperText = () => (
+  <span className="max-w-xl">
+    <i>
+      For additional information on where to submit, refer to the{" "}
+      {/* TODO: Get desired FAQ `target` from Wale */}
+      <Link
+        className="text-sky-600 hover:text-sky-800 underline"
+        target={"_faq"}
```

Comment:
Hey @Walesango2 -- what is the `target` value you're using for links to the FAQ? I wanna make sure this value matches so there's no need to go back and update this link. :)